src/main/java/utils/RandomUtils.java

```diff
@@ -0,0 +1,16 @@
+package utils;
+
+import java.util.Random;
+
+public class RandomUtils {
+    private static final Random random = new Random();
+    private static final int MAX_RANDOM_VALUE = 10;
+
+    private RandomUtils() {
+
+    }
```

Comment:
```suggestion
    private RandomUtils() {
    }
```

private 생성자를 통한 인스턴스화 방지 👍